tests/docker-tests/run-tests.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+# Copyright 2016 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+vicpath="$GOPATH/src/github.com/vmware/vic"
+
+if [ "$#" -gt "0" ]; then
+	if [ $1 = "clean" ];
+	then
+		cd ${vicpath}/tests/docker-tests
```

Comment:
Let's put all the copied files into their own sub-directory and add directory that to .gitignore too.